src/app.js

```diff
@@ -155,7 +156,13 @@ const App = ({ editorModel, output, configOpts }) => {
         />
         <Output
           orderClass={orderClass(OUTPUT)}
-          content={transformationError?.message || transformedOutput}
+          content={
+            transformationError
+              ? `${transformationError.message ?? "Error"}\n${
+                  transformationError.stack ?? ""
+                }`
+              : transformedOutput
+          }
```

Comment:
This is a nice addition! I would probably put the string builder in the `useTransformedOutput` function, but I think this is really just a matter of taste — I tend to avoid anything more complex than a single option in templates.